hazelcast-jet-kafka/src/main/java/com/hazelcast/jet/impl/connector/kafka/StreamKafkaP.java

```diff
@@ -254,8 +272,8 @@ public MetaSupplier(Properties properties, List<String> topicIds) {
             } else {
                 metadataRefreshInterval = KAFKA_DEFAULT_REFRESH_INTERVAL;
             }
-            // Set metadata caching to 1 second: we get metadata for multiple partitions one by one, but
-            // internally it is fetched at once. This enables for the other calls to just query fetched metadata.
+            // Set metadata caching to 1 second: we read the metadata for multiple partitions one by one. If we
```

Comment:
We should consistently apply the 72-char text line length rule even to these comments.